tests/python_functional/src/common/random_id.py

```diff
@@ -22,8 +22,10 @@
 #############################################################################
 import random
 
-UNIQUE_LENGTH = 5
+NUMBERS = list(range(10000, 99999))
+random.shuffle(NUMBERS)
 
 
-def get_unique_id(length=UNIQUE_LENGTH):
-    return str(random.getrandbits(16 * 8))[0:length]
+def get_unique_id():
```

Comment:
`port_allocator` is currently a fixture, which can only be used in the test cases. Do you suggest to move it to a separate module, like `random_id.py`? Or just copy its logic?

I did not want to lose the randomness factor, but if we can forget that, the always increasing logic is better, because it has no limit.

Reply:
Just copying the logic is fine.

Yeah, but this should provide unique, and random is for sure not unique. Sure picking randomly from unique items are unique, but why bother with randomness?
At least a larger limit :) which is trivial how to extend (meaning, it is well-defined what should be the next elem).

Reply:
16 bytes of random number should be unique in practice. Did you somehow run into collision?
Maybe use a uuid?

Unfortunately I do not remember why the original code was written this way.

Reply:
Yes, we have collision multiple times (in driver name in syslog-ng config).
I would not use uuid, it is too messy.

Reply:
Hmm interesting. This might worth investigating further.

16 bytes is a 128 bit number: 2^128 = 340282366920938463463374607431768211456
The winning chance of lottery (5 numbers out of 90) is 43949268.

There may be other kind of bug or design problem that causes the collision.

Reply:
`[0:length]` limits it to 5 bits unfortunately. Maybe we just need to remove that.

Reply:
Aah that explains it sorry.

Reply:
I fixed this with not restricting the length by default, but leaving the option to do so, if someone wants.